src/Plugin/Visualisation/Style/VisualisationStyleBase.php

```diff
@@ -132,7 +133,8 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
     $form['data']['fields'] = [
       '#type' => 'select',
       '#title' => $this->t('Fields'),
-      '#description' => $this->t("What fields to include in the visualisation. Select at least one field to display it's data. A field is typically a column in a CSV."),
+      '#description' => $this->t('What fields to include in the visualisation. Select at least one field to display its data. A field is typically a column in a CSV. @help',
+        ['@help' => \Drupal::service('dvf.helpers')->getHelpPageLink('keys')]),
```

Comment:
You have called `\Drupal::service('dvf.helpers')` 4 times in this file, it would be better to assign that to a variable eg `$dvf_helpers = \Drupal::service('dvf.helpers')` then use it like `$dvf_helpers->getHelpPageLink('keys')` This would improve both readability and performance

Reply:
Updated! :)